src/js/views/patients/sidebar/action-sidebar_activity-views.js

```diff
@@ -0,0 +1,105 @@
+    {{/with}}
+    <div>{{formatMoment date "LONG"}}</div>
+  `,
+});
+
+const ClinicianAssignedView = View.extend({
+  template: hbs`
+    {{#with metadata}}
+    {{formatMessage (intlGet "patients.sidebar.actionSidebar.activityViews.clinicianAssigned") first_name = editor.first_name last_name = editor.last_name role = editor.role.name to_first_name = value.to.first_name to_last_name = value.to.last_name}}
+    {{/with}}
+    <div>{{formatMoment date "LONG"}}</div>
+  `,
+});
+
+const DetailsUpdatedView = View.extend({
+  template: hbs`
+    {{#with metadata.editor}}
+    {{formatMessage (intlGet "patients.sidebar.actionSidebar.activityViews.detailsUpdated") first_name = first_name last_name = last_name role = role.name}}
+    {{/with}}
+    <div>{{formatMoment date "LONG"}}</div>
+  `,
+});
+
+const DueDateUpdatedView = View.extend({
+  template: hbs`
+    {{#with metadata.editor}}
+    {{formatMessage (intlGet "patients.sidebar.actionSidebar.activityViews.dueDateUpdated") first_name = first_name last_name = last_name role = role.name}}
+    {{/with}} {{formatMoment metadata.value.to "LONG"}}
+    <div>{{formatMoment date "LONG"}}</div>
+  `,
+});
+
+const DurationUpdatedView = View.extend({
+  template: hbs`
+    {{formatMessage (intlGet "patients.sidebar.actionSidebar.activityViews.durationUpdated") first_name = first_name last_name = last_name role = role duration = duration}}
+    <div>{{formatMoment date "LONG"}}</div>
+  `,
+  templateContext() {
+    const metadata = this.model.get('metadata');
+
+    return {
+      duration: duration(metadata.value.to).get('minutes'),
+      first_name: metadata.editor.first_name,
+      last_name: metadata.editor.last_name,
+      role: metadata.editor.role.name,
+    };
+  },
+});
+
+const NameUpdatedView = View.extend({
+  template: hbs`
+    {{#with metadata}}
+    {{formatMessage (intlGet "patients.sidebar.actionSidebar.activityViews.nameUpdated") first_name = editor.first_name last_name = editor.last_name role = editor.role.name to_name = value.to from_name = value.from}}
+    {{/with}}
+    <div>{{formatMoment date "LONG"}}</div>
+  `,
+});
+
+const RoleAssignedView = View.extend({
+  template: hbs`
+    {{#with metadata}}
+    {{formatMessage (intlGet "patients.sidebar.actionSidebar.activityViews.roleAssigned") first_name = editor.first_name last_name = editor.last_name role = editor.role.name to_role = value.to.name}}
+    {{/with}}
+    <div>{{formatMoment date "LONG"}}</div>
+  `,
+});
+
+const StateUpdatedView = View.extend({
+  template: hbs`
+    {{#with metadata}}
+    {{formatMessage (intlGet "patients.sidebar.actionSidebar.activityViews.stateUpdated") first_name = editor.first_name last_name = editor.last_name role = editor.role.name to_state = value.to.name}}
+    {{/with}}
+    <div>{{formatMoment date "LONG"}}</div>
+  `,
+});
+
+const ActivitiesView = CollectionView.extend({
+  childView(model) {
+    const ChildViews = {
+      ActionAdded: AddedView,
```

Comment:
I believe this is changing to `ActionCreated` https://app.clubhouse.io/roundingwell/story/20723/modify-action-search-filters#activity-20725